src/fsck/fsck.c

```diff
@@ -31,6 +31,7 @@
 #include "socket-util.h"
 #include "special.h"
 #include "stdio-util.h"
+#include "bus-locator.h"
```

Comment:
sort alphabetically within the same block

Reply:
Awesome sir.

Let me boot up my VM!